lib/circlex/api/payments/bank_accounts.ex

```diff
@@ -54,6 +56,47 @@ defmodule Circlex.Api.Payments.BankAccounts do
     end
   end
 
+  @doc """
+  Create a bank account (wires) for an international bank account that supports list_bank_accounts
```

Comment:
Do you have a reference here like `Reference: https://developers.circle.com/reference/payments-bank-accounts-wires-create` (from line 15 above). I tried to keep everything as 1:1 with Circle as possible, mostly because their APIs are so haphazard.

Reply:
FYI it's on the body params , anchor tags to link directly to it . Will make another note in the docstring though
![Screen Shot 2022-09-08 at 6 07 39 PM](https://user-images.githubusercontent.com/32463466/189234577-24924001-b118-464b-b852-73c2f9a89562.png)